src/app/component/shared/detail-view/view/view.component.ts

```diff
@@ -0,0 +1,107 @@
+import { Component, Input, OnInit } from "@angular/core";
+import { DateTime, Duration } from "luxon";
+import { toRelative } from "src/app/interfaces/apiInterfaces";
+
+@Component({
+  selector: "app-view",
```

Comment:
```suggestion
  selector: "render-field",
```

Reply:
Needs to have the `app` at the start to remove chance of collision with libraries. But `app-render-view` works.

Reply:
As per https://angular.io/guide/styleguide#component-custom-prefix yes it should be prefixed

but we should namespace it with our app identifier (not `app` that's too generic).